### PR TITLE
Default value fix

### PR DESCRIPTION
**Pull request by ilya-varchenya**

### CODE REVIEW

File: src/com/qaprosoft/jenkins/jobdsl/factory/pipeline/CronJobFactory.groovy [hunk 1]
```diff
@@ -49,7 +49,7 @@ public class CronJobFactory extends PipelineFactory {
                     choiceListProvider {
                         textareaChoiceListProvider {
                             choiceListText(getEnvironments(currentSuite))
-                            defaultChoice('')
+                            defaultChoice(getFirstEnvironmentsValue(currentSuite))
```
Comment (vdelendik) [inline src/com/qaprosoft/jenkins/jobdsl/factory/pipeline/CronJobFactory.groovy:52]: 1. let's rename getFirstEnvironmentsValue to getDefaultChoiceValue

File: src/com/qaprosoft/jenkins/jobdsl/factory/pipeline/PipelineFactory.groovy [hunk 2]
```diff
@@ -68,6 +68,11 @@ public class PipelineFactory extends JobFactory {
         return parsedEnviroments
     }
 
+    protected String getFirstEnvironmentsValue(currentSuite) {
+        def enviroments = getEnvironments(currentSuite)
+        return enviroments.substring(0, enviroments.indexOf('\n'))
```
Comment (vdelendik) [inline src/com/qaprosoft/jenkins/jobdsl/factory/pipeline/PipelineFactory.groovy:73]: does it support use-case when empty string as environments is provided?

Comment (ilya-varchenya) [reply src/com/qaprosoft/jenkins/jobdsl/factory/pipeline/PipelineFactory.groovy:73]: yes, getEnvironments() method support this functionality

File: src/com/qaprosoft/jenkins/jobdsl/factory/pipeline/CronJobFactory.groovy (hunk 1 above)
Comment (ilya-varchenya) [reply src/com/qaprosoft/jenkins/jobdsl/factory/pipeline/CronJobFactory.groovy:52]: renamed

**Pull request merged**